lxd/storage_volumes_backup.go

```diff
@@ -665,8 +664,8 @@ func storagePoolVolumeTypeCustomBackupPost(d *Daemon, r *http.Request) response.
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["volume"] = []string{volumeName}
```

Comment:
Lets keep volume category here, but use the "storage_volumes" category name like above, whilst also keeping the the "backups" category you've added here.